### DISCUSSION TITLE
Python code cannot replace span tag

### DISCUSSION THREAD

**Discussion by AbinReghunathan** [community/community#41909] [Projects and Issues]
req_data['Date']=req_data.Date.replace('<span class=','',inplace=True)
req_data['Date']

gives a None result

<html>
<body>

id | Reference_Name | Reference_Link | Date | Year
-- | -- | -- | -- | --
0 | General Python FAQ — Python 3.9.2 documentation | https://docs.python.org/3/faq/general.html#wha... | <span class="nowrap">28 March</span> | <span class="reference-accessdate">. Retrieved...
1 | Python 0.9.1 part 01/21 | https://www.tuhs.org/Usenet/alt.sources/1991-F... | <span class="nowrap">11 August</span> | <span class="reference-accessdate">. Retrieved...
2 | Python 3.11.1, 3.10.9, 3.9.16, 3.8.16, 3.7.16,... | https://pythoninsider.blogspot.com/2022/12/pyt... | <span class="nowrap">7 December</span> | <span class="reference-accessdate">. Retrieved...
3 | Why is Python a dynamic language and also a st... | https://wiki.python.org/moin/Why%20is%20Python... | <span class="nowrap">27 January</span> | <span class="reference-accessdate">. Retrieved...
4 | PEP 483 – The Theory of Type Hints | https://www.python.org/dev/peps/pep-0483/ | <span class="nowrap">14 June</span> | <span class="reference-accessdate">. Retrieved...

</body>
</html>

Unable to remove the span tag from the scraped data.

**Comment by DSDmark** [comment]
Hi @AbinReghunathan,

Change your category into **issue**.

Can you **specific more** about the details of what you want? For better understanding 😅. Also, you can add some **screenshots** or **error** which you were facing 👍.

**Comment by ossanna16** [comment]
Hi @AbinReghunathan,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Python. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
